parameter-sets/node-pool-request/parameter-set.json

```diff
@@ -156,6 +156,18 @@
             "type": "KEY_VALUE_LIST",
             "label": "K8S labels",
             "description": "Custom K8S metadata to attach to each node"
+        },
+        {
+            "name": "nodepoolTaints",
```

Comment:
Added it and successfully scaled up from 0:
<img width="2225" alt="image" src="https://github.com/user-attachments/assets/22bf870d-61a9-4143-951c-893711c0646e">

I have not tested with GPUs since they are not available in europe-west9-a: https://cloud.google.com/compute/docs/gpus/gpu-regions-zones#view-using-table